recipes/cc65/all/conanfile.py

```diff
@@ -0,0 +1,175 @@
+            prefix = tools.unix_path(prefix)
+            samplesdir = tools.unix_path(samplesdir)
+        return [
+            "PREFIX={}".format(prefix),
+            "datadir={}".format(datadir),
+            "samplesdir={}".format(samplesdir),
+        ]
+
+    def _build_autotools(self):
+        autotools = self._configure_autotools()
+        with tools.chdir(os.path.join(self._source_subfolder)):
+            autotools.make(args=self._make_args)
+
+    def _patch_sources(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+        if self.settings.compiler == "Visual Studio":
+            with tools.chdir(os.path.join(self._source_subfolder, "src")):
+                for fn in os.listdir("."):
+                    if not fn.endswith(".vcxproj"):
+                        continue
+                    tools.replace_in_file(fn, "v141", msvs_toolset(self.settings))
+                    tools.replace_in_file(fn, "<WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>", "")
+
+    def build(self):
+        self._patch_sources()
+        with self._mock_settings():
+            if self.settings.compiler == "Visual Studio":
+                self._build_msvc()
+            else:
+                self._build_autotools()
+
+    def _package_msvc(self):
+        self.copy("*.exe", src=os.path.join(self._source_subfolder, "bin"), dst=os.path.join(self.package_folder, "bin"), keep_path=False)
+        for dir in ("asminc", "cfg", "include", "lib", "target"):
+            self.copy("*", src=os.path.join(self._source_subfolder, dir), dst=os.path.join(self._datadir, dir))
+
+    def _package_autotools(self):
+        autotools = self._configure_autotools()
+        with tools.chdir(os.path.join(self.build_folder, self._source_subfolder)):
+            autotools.install(args=self._make_args)
+
+        tools.rmdir(self._samplesdir)
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        if self.settings.compiler == "Visual Studio":
+            self._package_msvc()
+        else:
+            self._package_autotools()
+
+    def package_id(self):
+        self.info.include_build_settings()
+        if self.settings.compiler == "Visual Studio":
+            if self.settings.arch_build == "x86_64":
+                self.info.settings.arch_build = "x86"
+        del self.info.settings.compiler
+
+    def package_info(self):
+        bindir = os.path.join(self.package_folder, "bin")
+        self.output.info("Appending PATH environment variable: %s" % bindir)
+        self.env_info.PATH.append(bindir)
+
+        self.output.info("Seting CC65_HOME environment variable: %s" % self._datadir)
+        self.env_info.CC65_HOME = self._datadir
+
+        cc65_cc = os.path.join(bindir, "cc65")
+        if self.settings.os_build == "Windows":
```

Comment:
this repeats several time, so maybe add variable like:
```
ext = ".exe" if self.settings.os_build == "Windows" else ""
```
then:
```
cc65_cc = os.path.join(bindir, "cc65") + ext
cc65_as = os.path.join(bindir, "ca65") + ext
cc65_ld = os.path.join(bindir, "cl65") + ext
```

Reply:
Great simplification. Love it!